frontend/packages/container-security/src/components/image-manifest-vuln.tsx

```diff
@@ -302,7 +302,7 @@ export const ImageManifestVulnList: React.FC<ImageManifestVulnListProps> = (prop
 };
 
 export const ImageManifestVulnPage: React.FC<ImageManifestVulnPageProps> = (props) => {
-  const namespace = props.match.params?.ns;
+  const namespace = props.match?.params?.ns;
```

Comment:
This fixes the runtime error, but the list will presumably show vulnerabilities for *all projects*, not just the current project. We should compare this to other list views to see what's different. I believe the namespace is passed through in a different way for other pages.

Reply:
I looked through the other components using MultiListPage and they're either getting the namespace passed in as a prop directly or using props.match.params.ns.